code/controllers/subsystem/mapping.dm

```diff
@@ -239,7 +239,7 @@ SUBSYSTEM_DEF(mapping)
 
 	// load mining
 	if(config.minetype == "lavaland")
-		LoadGroup(FailedZs, "Lavaland", "map_files/Mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND)
+		LoadGroup(FailedZs, "Lavaland", "yogstation/map_files/mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND)
```

Comment:
reeeeeeeeee put a //yogs you fudge

Reply:
oh yeah, I've beeen doing non-yogs stuff and forgot

Reply:
>:(